src/js/api.ts

```diff
@@ -51,6 +52,10 @@ API.saveState = () => API.pyodide_py._state.save_state();
 /** @private */
 API.restoreState = (state: any) => API.pyodide_py._state.restore_state(state);
 
+// Used in webloop
+/** @private */
+API.scheduleCallback = scheduleCallback;
```

Comment:
You can just attach it to `API` in `scheduler.ts`.

Reply:
It didn't work because then `scheduler.ts` is not imported from anywhere. Let me merge this as-is and the revise the code structure later. Thanks for the suggestion!